subiquity/ui/views/drivers.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2021 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+""" Module defining the view for third-party drivers installation.
+
+"""
+import asyncio
+import logging
+from typing import List, Optional
+
+from urwid import (
+    connect_signal,
+    Text,
+    )
+
+from subiquitycore.ui.buttons import ok_btn
+from subiquitycore.ui.form import (
+    Form,
+    BooleanField,
+)
+from subiquitycore.ui.spinner import Spinner
+from subiquitycore.ui.utils import screen
+from subiquitycore.view import BaseView
+
+
+log = logging.getLogger('subiquity.ui.views.drivers')
+
+
+class DriversForm(Form):
+    """ Form that shows a checkbox to configure whether we want to install the
+    available drivers or not. """
+
+    cancel_label = _("Back")
+
+    install = BooleanField(_("Install the drivers"))
+
+
+class DriversView(BaseView):
+
+    title = _("Third-party drivers.")
```

Comment:
The other titles don't have a period
```suggestion
    title = _("Third-party drivers")
```